ethereum-testnets.asciidoc

```diff
@@ -1,34 +1,49 @@
 
+[TIP]
+====
+Test ether is specific to each testnet. You can use the same account on multiple networks, even across the Ethereum main network and various public testnets. However, on each network, the balance of your account is tracked independently and if you have ether on one network, it does not appear on any other network. To use multiple testnets you must load your account with test ether on each network.
+====
+
 === Connecting to Testnets
 
 ==== Metamask
-Metamask fully supports the Ropsten, Kovan and Rinkeby testnets, but connecting to other testnets and local networks is also possible. In Metamask, simpy clicking the drop down that says 'Main Network' allows for switching networks. MetaMask offers an option to buy ether. If the Ropsten testnet is used, ether can be obtained from the Ropsten Test Faucet Service. You can access this faucet from this page. It requires the Metamask extension to work. https://faucet.metamask.io/
+Metamask fully supports the Ropsten, Kovan and Rinkeby testnets, but connecting to other testnets and local networks is also possible. In Metamask, simpy clicking the drop down that says 'Main Network' allows you to switch networks. MetaMask also offers an option to "buy" test ether,  . If the Ropsten testnet is used, ether can be obtained from the Ropsten Test Faucet Service. You can access this faucet from this page. It requires the Metamask extension to work. https://faucet.metamask.io/
+
 
-==== Infura
+===== Infura
 
-Infura was born with the goal of delivering stable and reliable RPC access to the internal projects within ConsenSys. As the Ethereum network began to grab the majority of blockchain developer mindshare, ConsenSys recognized that Infura would be of broad interest to the entire ecosystem. It provides access to Ethereum APIs and IPFS gateways. Using Infura with testnets is simple enough. Infura supports gatweays for the main net, Ropsten, Kovan, Rinkeby and INFURAnet (a custom testnet for Infura). Sending API calls to testnets from Infura simply involves changing the base URL for API calls.  More information on the details of these API calls is located at:
-https://infura.io/docs/gettingStarted/chooseaNetwork
+When MetaMask connects to a test network, it uses the Infura service provider for the JSON-RPC interface. Infura was born with the goal of delivering stable and reliable RPC access to the internal projects within ConsenSys. In addition to a JSON-RPC API, Infura also offers a REST (Representational State Transfer) API, IPFS (Interplanetary File System, ie. decentralized storage) API, and a Websockets (ie. streaming) API.
+
+Infura offers gateway APIs to the Ethereum main net, Ropsten, Kovan, Rinkeby, and INFURAnet (a custom testnet for Infura).
```

Comment:
a note about the security implications of a trusted RPC endpoint (on the mainnet especially) would be good; lots of new users wouldn't be aware how much this increases the trust requirement.

Reply:
:+1:  @rphmeier